packages/app/src/cli/services/logs.ts

```diff
@@ -61,28 +64,35 @@ export async function logs(commandOptions: LogsOptions) {
   }
 
   if (commandOptions.format === 'json') {
+    consoleLog(JSON.stringify({subscribedToStores: commandOptions.storeFqdns}))
     consoleLog(JSON.stringify({message: 'Waiting for app logs...'}))
     await renderJsonLogs({
       options: {
         variables,
         developerPlatformClient: logsConfig.developerPlatformClient,
       },
       pollOptions,
+      storeNameById: logsConfig.storeNameById,
     })
   } else {
+    if (multipleStores) {
+      consoleLog(`Subscribing to additional stores: ${commandOptions.storeFqdns?.slice(1).join(', ')}\n`)
```

Comment:
The first shop will display as part of the `info` box rendered by `ensureDevContext` in a TTY, so we only need to call out the additional stores.

Not sure if we want this at verbose level instead.